scriptmodules/emulators/mupen64plus.sh

```diff
@@ -330,29 +340,35 @@ function configure_mupen64plus() {
         su "$user" -c "$cmd"
     fi
 
-    # RPI main/GLideN64 settings
+    iniConfig " = " "" "$config"
+    # RPI main settings
     if isPlatform "rpi"; then
-        iniConfig " = " "" "$config"
         # VSync is mandatory for good performance on KMS
         if isPlatform "kms"; then
             if ! grep -q "\[Video-General\]" "$config"; then
                 echo "[Video-General]" >> "$config"
             fi
             iniSet "VerticalSync" "True"
         fi
-        # Create GlideN64 section in .cfg
-        if ! grep -q "\[Video-GLideN64\]" "$config"; then
-            echo "[Video-GLideN64]" >> "$config"
-        fi
-        # Settings version. Don't touch it.
-        iniSet "configVersion" "29"
+    fi
+
+    # Create GlideN64 section in .cfg
+    if ! grep -q "\[Video-GLideN64\]" "$config"; then
+        echo "[Video-GLideN64]" >> "$config"
+    fi
+    # Settings version
+    iniSet "configVersion" "$gliden64_ver"
+    # Use native res
+    iniSet "UseNativeResolutionFactor" "1"
+    # Enable legacy blending
+    iniSet "EnableLegacyBlending" "True"
```

Comment:
Please remove this line. EnableLegacyBlending helps to get some extra fps on slow devices but it can cause visual problems. If "mupen64plus_compatibility_check" autoconf is set it will be enabled anyway.

Reply:
In the previous commit EnableLegacyBlending was only true for rpi: https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/emulators/mupen64plus.sh#L334-L355

I think that's right as all (current) pis are fighting for performance in GLideN64, and the blacklist will disable it for known problem games. I am not sure why it (and some other settings) been taken out of that block - perhaps unintentional @joolswills  ? :)

however you do raise an issue with the launch script, which seems to always enable, and the disable it if the game is in the blacklist. IMO that's wrong - it should just disable when necessary rather than ignore the cfg setting. i.e. remove this line: https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/emulators/mupen64plus/mupen64plus.sh#L267

Reply:
@dankcushions it just resets to best „rpi“ default setting for performance before it checks blacklist. Autoset function could be the better location. If you want to preserve user settings you need to read,  store and restore it after m64p closes.

Reply:
I'll change it. I got the wrong default (probably confused due to the compatibility check code). I wanted to make sure we always have the keys that are to be used by the launch script which is why I added them outside.

Reply:
Yeah it's a little unwieldy! https://github.com/RetroPie/RetroPie-Setup/pull/2960 simplifies this some ;) it adds the headers if they don't exist.